crates/subspace-networking/src/bin/subspace-bootstrap-node/main.rs

```diff
@@ -8,8 +8,10 @@ use subspace_networking::libp2p::multiaddr::Protocol;
 use subspace_networking::{BootstrappedNetworkingParameters, Config};
 use tracing::info;
 
-// The default maximum connection (in and out) number for the peer.
-const MAX_ESTABLISHED_CONNECTIONS: u32 = 5000;
+// The default maximum incoming connections number for the peer.
+const MAX_ESTABLISHED_INCOMING_CONNECTIONS: u32 = 5000;
+// The default maximum outgoing connections number for the peer.
+const MAX_ESTABLISHED_OUTGOING_CONNECTIONS: u32 = 5000;
```

Comment:
Outgoing shouldn't default to 5000. It is the target node is striving for, but we certainly don't want it to try and establish that many connections proactively by default.